gdnative-core/src/core_types/vector2.rs

```diff
@@ -89,6 +90,17 @@ impl Vector2Godot for Vector2 {
         Vector2::new(self.y, -self.x)
     }
 
+    #[inline]
+    fn move_towards(self, to: Vector2, delta: f32) -> Self {
+        let vd = to - self;
+        let len = vd.length();
+        if len <= delta || approx::abs_diff_eq!(len, 0.00001) {
```

Comment:
I don't think you need to manually specify this epsilon value here. Also, you'll need to change `approx` from an optional dependency to a required one, since it's now used in the main library code, not just in tests.

Reply:
changed to use f32::EPSILON, how to change `approx` to be dependency? i tried change `approx = { version = "0.3.2", optional = true }`  to `approx = "0.3.2"` in cargo.toml but it says:
 ``error: failed to parse manifest at `/home/karpiu/Godot Engine/godot-rust/gdnative-core/Cargo.toml``
`Caused by:`
`Feature `gd_test` depends on `approx` which is not an optional dependency.`
`Consider adding ``optional = true` to the dependency``

Reply:
Remove `approx` from the `gd_test` feature.

Reply:
thanks, changed